webpack/InsightsCloudSync/InsightsCloudSyncActions.js

```diff
@@ -0,0 +1,20 @@
+import API from 'foremanReact/API';
+import { insightsCloudUrl } from './InsightsCloudSyncHelpers';
+import { INSIGHTS_CLOUD_SYNC_TASK } from './InsightsCloudSyncConstants';
+
+export const syncInsights = () => async dispatch => {
+  try {
+    await API.post(insightsCloudUrl('tasks'));
+    dispatch({
+      type: INSIGHTS_CLOUD_SYNC_TASK,
+      payload: {},
+    });
+  } catch (error) {
+    dispatch({
+      type: INSIGHTS_CLOUD_SYNC_TASK,
```

Comment:
we should distinguish between the `"succes"/"error"` responses
can you rename them to something like: `INSIGHTS_SYNC_SUCCESS`, `INSIGHTS_SYNC_FAILURE`?